### DISCUSSION TITLE
New release of main.js code on bitclout tonight

### DISCUSSION THREAD

**Discussion by scottstirling** [scottstirling/bitcloutDAO#23] [General]
Just FYI, minutes ago - updated here now as well.

Will do some reorg around license files, README.md's and structure ASAP

**Comment by scottstirling** [comment]
One new feature is pre-populating the Help email to node.admin@bitclout.com with details in the message body (actual key and address removed from example below):

```The below information helps support address your case.
My public key: <pre-filled public BitClout key> 
My BTC Address: <pre-filled Bitcoin wallet address>

**Reply by ianhernandez** [reply]
awesome nice find scott
